### PR TITLE
Feature/copy

### PR DESCRIPTION
**Pull request by Alexintosh**
##### Description

This PR changes some of the copy and language about the explanation of PIE.

##### Checklist

- [x] Changes don't break existing behavior
- [ ] All content should be hypothetical
- [ ] Check grammar
- [ ] Make sure I didn't say anything inappropriate

### CODE REVIEW

File: src/Components/GradienBgSection.js [hunk 1]
```diff
@@ -45,13 +48,17 @@ const GradienBgSection = props => {
...
+          <strong>Top investment deals are impossible to get for the most of us.</strong>
+          <br></br><br></br>
 
-We use a combination of derivatives, synthetic assets and cryptocurrencies to mirror battle proof hedge funds.<p></p>
-<A href="#">Lightpaper</A>
-  </Text>
+          For example, Ray Dalio used to accept money from investors who had a net worth of at least <strong>$5 billion</strong> and with a minimum investment requirement of <strong>$100 million</strong>. 
+          <br></br>Nowadays he won’t accept any new investors.
+          <br></br><br></br>
+          We are making the best asset allocation strategies accessible to everyone on the Ethereum network with no minimum.
```
Comment (MickdeGraaf) [inline src/Components/GradienBgSection.js]: This claim is quite bold. A strategy that has historically good performance is not guaranteed to have the "best" performance when the economical climate changes

File: src/Components/FAQ.js [hunk 2]
```diff
@@ -62,118 +62,225 @@ const FAQ = props => {
         <Gradient>
           <SingleItem>
             <Collapsible
-              trigger="Who the hell is Ray Dalio?"
+              trigger="What is Pie?"
```
Comment (MickdeGraaf) [inline src/Components/FAQ.js]: Would suggest to change it to "What is the Pie Network". This avoids confusion with the token which ticker will be PIE

File: src/Components/FAQ.js [hunk 3]
```diff
@@ -62,118 +62,225 @@ const FAQ = props => {
...
-              trigger="Is it safe?"
+              trigger="Which protocol are used inside the Pie Network smart contracts?"
               easing="ease-in"
               transitionTime={100}
               triggerTagName="div"
             >
               <p>
-                This is the collapsible content. It can be any element or React
-                component you like.
+              Pie Dao uses existing DeFi protocols as building blocks to unlock the compounding value of multiple financial primitives. Pie Network combines the use of synthetic asset platforms, and decentralized exchanges.
```
Comment (MickdeGraaf) [inline src/Components/FAQ.js]: Maybe use "The Pie Dao" consistently. With "the" and consistent capitalisation of Dao vs DAO

File: src/Components/FAQ.js [hunk 4]
```diff
@@ -62,118 +62,225 @@ const FAQ = props => {
...
+              trigger="What's different from Compound?"
               easing="ease-in"
               transitionTime={100}
               triggerTagName="div"
             >
               <p>
-                This is the collapsible content. It can be any element or React
-                component you like.
+                Compound is a transparent, autonomous money market— allowing users & applications to frictionlessly earn interest or borrow Ethereum assets without relying on a counterparty.
+                Interest for lenders on Compound are calculated algorithmically at every block, therefore, having predictability on ROI by holding cTokens is challending.
```
Comment (MickdeGraaf) [inline src/Components/FAQ.js]: "The interest rate for lenders is calculated"

File: src/Components/FAQ.js [hunk 5]
```diff
@@ -62,118 +62,225 @@ const FAQ = props => {
...
-              trigger="How can I invest?"
+              trigger="Is PIE itself a derivative?"
               easing="ease-in"
               transitionTime={100}
               triggerTagName="div"
             >
               <p>
-                This is the collapsible content. It can be any element or React
-                component you like.
+                PIE is not a derivative, it is a protocol for cryptocurrency users to create their own tokenized asset allocation using sythetics assets. PIE is a set of open source smart contracts that can be deployed to the Ethereum blockchain. The Pie Network is nothing more than open source software, and individual users of the Pie Network are the ones that create a tokenized derivative using the Pie Network. The Pie DAO has zero involvement in the creation of individual derivative tokens on the Pie Network, and the Pie DAO is not "Pie Network".
```
Comment (MickdeGraaf) [inline src/Components/FAQ.js]: To me PIE refers to the token and not the network

File: src/Components/FAQ.js (hunk 4 above)
Comment (MickdeGraaf) [inline src/Components/FAQ.js]: challenging typo

File: src/Components/FAQ.js (hunk 5 above)
Comment (MickdeGraaf) [inline src/Components/FAQ.js]: deployed on vs deployed to the Ethereum network

**Comment by Alexintosh** [comment]
Thanks for the comments, I will take those into consideration and fix the conflicts.

**Pull request merged**